### ISSUE TITLE
Introduce custom command for license checker

### ISSUE THREAD

**Issue by kelsos** [raiden-network/light-client#1739]
Nitpick:
We have basically the same line 3 times with the same comment copied. Since this is a mono repo we can't know if there isn't more to come. But even these three are enough in my opinion to make this somewhat re-usable. If you agree and like the idea I would recommand to create a [custom command](https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-commands) and add the directory as a parameter for this command. This approach might be nice for some other places as well. :shrug:

_Originally posted by @weilbith in https://github.com/_render_node/MDI0OlB1bGxSZXF1ZXN0UmV2aWV3Q29tbWVudDQ0MjA3NzgyNQ==/comments/review_comment_

**Comment by kelsos** [comment]
@weilbith if you want to expand on this please feel free to edit the description with more details.

**Comment by weilbith** [comment]
Not really. I think you did a good job in also linking the conversation of the PR.
We should just checkout if there are maybe more common parts between the three scopes of this repo. I'm not familiar with the configuration yet. But I think this can be simply done by a single person on its own without much discussion. It is a simple optimization for the human reading/writing in first place. So nothing critical.

**Comment by kelsos** [comment]
You can actually create an issue from a comment using the overflow menu. All of this was automatic I just pressed a button :D
